pyproject.toml

```
@@ -21,7 +21,7 @@ kafka-python = "^2.0.1"
 pydantic = "^1.5.1"
 orjson = "^3.3.1"
 jsonschema = "^3.2.0"
-prometheus_client = "^0.8.0"
+prometheus_client = ">= 0.8.0"
```

I think this is what we need: 
```suggestion
prometheus_client = "^0.8"
```